src/main/java/spoon/support/compiler/jdt/JDTBasedSpoonCompiler.java

```diff
@@ -144,6 +145,13 @@ public boolean compile(InputType... types) {
 
 		JDTBatchCompiler batchCompiler = createBatchCompiler(types);
 
```

Comment:
Why does compile take an array if the method support only arrays of length 1?

Reply:
Les voies du seigneur sont impénétrables.